ui/components/component-library/text/text.js

```diff
@@ -154,5 +154,3 @@ Text.propTypes = {
 };
 
 Text.displayName = 'Text'; // Used for React DevTools profiler
-
-export default Text;
```

Comment:
This should probably be apart of `Text` housekeeping but not blocking as these are small simple updates